docs/conf.py

```diff
@@ -15,20 +15,22 @@
 import os
 import sys
 
+import fixate
```

Comment:
Pretty sure this import needs to be after the `sys.path.append` otherwise it won't be able to find the fixate module.

Reply:
🤦‍♂️ this is why one _automates_ things, right? 😉